### PR TITLE
New Patient field group

### PR DESCRIPTION
**Pull request by amadobe**
[PLAT-192386](https://jira.corp.adobe.com/browse/PLAT-192386)

### CODE REVIEW

File: extensions/industry/healthcare/fhir/fieldgroups/patient.schema.json [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+   "$schema":"http://json-schema.org/draft-06/schema#",
+   "title":"Patient",
+   "type":"object",
+   "meta:tags":{
+      "industry":[
+         "healthcare_services"
+      ]
+   },
+   "meta:sensitive-data":true,
+   "description":"Demographics and other administrative details about an individual or animal receiving care or other health-related services.",
```
Comment (anandphatak) [inline extensions/industry/healthcare/fhir/fieldgroups/patient.schema.json:18]: We can add the line at the end o the description: "(Based on HL7 FHIR Release 5 specifications)"

File: extensions/industry/healthcare/fhir/fieldgroups/patient.schema.json [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+               "title":"Birth Number",
+               "type":"integer",
+               "description":"Birth number in the sequence."
+            },
+            "xdm:photo":{
+               "title":"Patient Photo",
+               "description":"Pateint's photogragh",
+               "type":"array",
+               "items":{
+                  "$ref":"#/definitions/image-or-link-or-array-of-image-or-link"
```
Comment (anandphatak) [inline extensions/industry/healthcare/fhir/fieldgroups/patient.schema.json:113]: This reference seems wrong

File: extensions/industry/healthcare/fhir/fieldgroups/patient.schema.json [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+                        "description":"The type of link between this patient resource and another patient resource.",
+                        "type":"string"
+                     }
+                  }
+               }
+            }
+         }
+      }
+   },
+   "allOf":[
```
Comment (anandphatak) [inline extensions/industry/healthcare/fhir/fieldgroups/patient.schema.json:231]: You will also need to include all datatype references used in this file into the "allOf" section, for example: 
"allOf": {
    {
        "$ref":"https://ns.adobe.com/xdm/industry/healthcare/fhir/datatypes/identifier"
    },
    {
         "$ref":"https://ns.adobe.com/xdm/industry/healthcare/fhir/datatypes/humanname"
    },
... and so on

File: extensions/industry/healthcare/fhir/fieldgroups/patient.schema.json (hunk 2 above)
Comment (amadobe) [reply extensions/industry/healthcare/fhir/fieldgroups/patient.schema.json]: removed the field. need to find a new reference data type

**Review by anandphatak — APPROVED** [review]

**Pull request merged**